gentoo/gentoo
# dev-python/backports-functools-lru-cache: bump to 1.3

https://github.com/jaraco/backports.functools_lru_cache/compare/1.2.1...1.3
```diff
--- backports-functools-lru-cache-1.2.1.ebuild  2016-10-04 11:38:59.113662103 +0200
+++ backports-functools-lru-cache-1.3.ebuild    2016-10-04 11:38:59.113662103 +0200
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 PYTHON_COMPAT=( python2_7 )
 
@@ -17,10 +17,15 @@
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="test"
+IUSE="doc test"
 
-DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+DEPEND="
+       dev-python/setuptools[${PYTHON_USEDEP}]
        >=dev-python/setuptools_scm-1.9[${PYTHON_USEDEP}]
+       doc? (
+               dev-python/sphinx[${PYTHON_USEDEP}]
+               dev-python/rst-linker[${PYTHON_USEDEP}]
+       )
        test? (
                >=dev-python/pytest-2.8[${PYTHON_USEDEP}]
                dev-python/pytest-runner[${PYTHON_USEDEP}]
@@ -29,13 +34,15 @@
 
 S="${WORKDIR}/${MY_PN}-${PV}"
 
-python_prepare_all() {
-       if use test && has_version "${CATEGORY}/${PN}"; then
-               die "Ensure $PN is not already installed or the test suite will fail"
-       fi
-       distutils-r1_python_prepare_all
+python_compile_all() {
+       use doc && esetup.py build_sphinx
 }
 
 python_test() {
        PYTHONPATH=. py.test || die "tests failed with ${EPYTHON}"
 }
+
+python_install_all() {
+       use doc && local HTML_DOCS=( "${BUILD_DIR}"/sphinx/html/. )
+       distutils-r1_python_install_all
+}
```
